designs/ClientDevProcess.md

```diff
@@ -0,0 +1,145 @@
+# NuGet Client Team Dev Process
+
+- Status: **Reviewing**
+- Author: [Kat Marchan](https://github.com/zkat)
+
+# Issue
+
+[#8994](https://github.com/NuGet/Home/issues/8994)
+
+# Background
+
+For a while now, the NuGet Client team has been using [ZenHub](https://zenhub.com) as part of its development process. At the same time, while ZenHub provides a lot of features, the team isn't taking advantage of some of the nicer ones, such as its charting capabilities and its Roadmap feature. It also has other features, like Blockers and Epics, which the team does use but could be using more often, or differently.
+
+This proposal is aimed at overhauling the Client Team's usage of ZenHub and further streamline its public development process to increase velocity, as well as increase visibility into what's being done, and what needs to be done. It also describes existing processes, so this document may be used as a reference for development flow.
+
+# Proposal
+
+## OKR Tracking
+
+- Link: https://github.com/NuGet/Home#workspaces/nuget-client-team-55aec9a240305cf007585881/roadmap?repos=29996513
+
+One of the main additions in this proposal is that it adds explicit tracking of OKRs into our ZenHub process. The way this works is by using ZenHub's Roadmap feature.
+

```

Comment:
I should have been clearer. I meant KR like  `M.n` form but I feel that will be too much of book keeping. I like the current approach of linking them to Objectives.